### PR TITLE
Chore: Added article links in italian footer

### PR DESCRIPTION
**Pull request by alevanni**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: client/i18n/locales/italian/trending.json [hunk 1]
```diff
@@ -1,62 +1,61 @@
...
+"article2link":"https://www.freecodecamp.org/italian/news/scorri-la-pagina-verso-lalto-come-scorrere-a-una-sezione-particolare-con-react/",
+"article3title":"Recuperare i dati in React",
+"article3link":"https://www.freecodecamp.org/italian/news/come-recuperare-i-dati-in-react-cheat-sheet-esempi/",
+"article4title":"Figure in CSS spiegate",
+"article4link":"https://www.freecodecamp.org/italian/news/figure-in-css-spiegate/"
+"article5title":"Linguaggi interpretati e compilati:",
+"article5link":"https://www.freecodecamp.org/italian/news/linguaggi-di-programmazione-interpretati-e-compilati-qual-e-la-differenza/",
+"article6title":"L'Hook useEffect di React",
+"article6link":"https://www.freecodecamp.org/italian/news/hook-useeffect-react/",
+"article7title":"Donare un milione di dollari a freeCodeCamp",
```
Comment (ieahleen) [inline client/i18n/locales/italian/trending.json:16]: questo forse è troppo lungo, il massimo sono 28 caratteri

File: client/i18n/locales/italian/trending.json [hunk 2]
```diff
@@ -1,62 +1,61 @@
...
+"article25title":"Metodo POST in JQuery Ajax",
+"article25link":"https://www.freecodecamp.org/italian/news/metodo-jquery-ajax-post/",
+"article26title":"Come commentare i JSON",
+"article26link":"https://www.freecodecamp.org/italian/news/esempio-di-commento-json-come-commentare-nei-file-json/",
+"article27title":"JavaScript Date Now",
+"article27link":"freecodecamp.org/italian/news/javascript-date-now-come-ottenere-la-data-corrente-in-javascript/",
+"article28title":"Elenco Keycode in JavaScript",
+"article28link":"https://www.freecodecamp.org/italian/news/elenco-codici-chiave-javascript/",
+"article29title":"Callback in JavaScript",
+"article29link":"https://www.freecodecamp.org/italian/news/funzioni-di-callback-javascript-cosa-sono-le-callback-in-js-e-come-usarle/"}
```
Comment (ieahleen) [inline client/i18n/locales/italian/trending.json:61]:
```suggestion
"article29link":"https://www.freecodecamp.org/italian/news/funzioni-di-callback-javascript-cosa-sono-le-callback-in-js-e-come-usarle/"
}

```

File: client/i18n/locales/italian/trending.json [hunk 3]
```diff
@@ -1,62 +1,62 @@
...
+"article0title":"Cambiare nome su Gmail",
+"article0link":"https://www.freecodecamp.org/italian/news/cambiare-nome-su-gmail/",
+"article1title":"Come creare un NFT",
+"article1link":"https://www.freecodecamp.org/italian/news/come-creare-un-nft-e-renderlo-su-opensea-marketplace/",
+"article2title":"Come scorrere con React",
+"article2link":"https://www.freecodecamp.org/italian/news/scorri-la-pagina-verso-lalto-come-scorrere-a-una-sezione-particolare-con-react/",
+"article3title":"Recuperare i dati in React",
+"article3link":"https://www.freecodecamp.org/italian/news/come-recuperare-i-dati-in-react-cheat-sheet-esempi/",
+"article4title":"Figure in CSS spiegate",
+"article4link":"https://www.freecodecamp.org/italian/news/figure-in-css-spiegate/"
```
Comment (ieahleen) [inline client/i18n/locales/italian/trending.json:11]:
```suggestion
"article4link":"https://www.freecodecamp.org/italian/news/figure-in-css-spiegate/",
```

Comment (ieahleen) [inline client/i18n/locales/italian/trending.json:11]:
```suggestion
"article4link":"https://www.freecodecamp.org/italian/news/figure-in-css-spiegate/",
```

**Review by raisedadead — DISMISSED** [review]

File: client/i18n/locales/italian/trending.json [hunk 4]
```diff
@@ -1,62 +1,62 @@
...
+  "article23title": "C vs C++",
+  "article23link": "https://www.freecodecamp.org/italian/news/c-vs-cpp-qual-e-la-differenza/",
+  "article24title": "Portfolio HTML, CSS e JS",
+  "article24link": "https://www.freecodecamp.org/italian/news/come-creare-un-sito-web-portfolio-utilizzando-html-css-javascript-e-bootstrap-5/",
+  "article25title": "Metodo POST in JQuery Ajax",
+  "article25link": "https://www.freecodecamp.org/italian/news/metodo-jquery-ajax-post/",
+  "article26title": "Come commentare i JSON",
+  "article26link": "https://www.freecodecamp.org/italian/news/esempio-di-commento-json-come-commentare-nei-file-json/",
+  "article27title": "JavaScript Date Now",
+  "article27link": "freecodecamp.org/italian/news/javascript-date-now-come-ottenere-la-data-corrente-in-javascript/",
```
Comment (nhcarrigan) [inline client/i18n/locales/italian/trending.json:57]:
```suggestion
  "article27link": "https://www.freecodecamp.org/italian/news/javascript-date-now-come-ottenere-la-data-corrente-in-javascript/",
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Review by majestic-owl448 — APPROVED** [review]

**Pull request merged**